config/versions.yml

```diff
@@ -92,3 +101,43 @@ versioning_systems:
   edot_cf_aws:
     base: 0.1
     current: 0.2.0
+  
+  # Logging
```

Comment:
@shainaraskas @colleenmcginnis We can add UpdateCLI automated version bumps for these, too, in a follow-up PR.